DigitalLibrary/DigitalLibrary/MainScreens/BookDetailsScreen/BookDetailsView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  BookDetailsView.swift
+//  DigitalLibrary
+//
+//  Created by Vasilka Terzieva on 16.02.24.
+//
+
+import SwiftUI
+
+struct BookDetailsView: View {
+    @ObservedObject var viewModel: BookDetailsViewModel
+
+    var body: some View {
+        ScrollView {
+            Image(viewModel.photo)
+                .resizable()
+                .scaledToFit()
+                .cornerRadius(12)
+                .shadow(radius: 5)
+                .padding(.horizontal, 80)
+                .padding(.vertical, 25)
+            
+            VStack(spacing: 8) {
+                Text(viewModel.title)
+                    .font(.title)
+                    .fontWeight(.semibold)
+                    .multilineTextAlignment(.center)
+                
+                Text(viewModel.author)
+                    .font(.headline)
+                    .foregroundColor(.gray)
+                
+                Text("\(viewModel.publisher) \(viewModel.year)")
+                    .font(.subheadline)
+                    .foregroundColor(.gray)
+                
+                Text(viewModel.description)
+                    .font(.body)
+                    .multilineTextAlignment(.center)
+                    .padding(.horizontal)
+            }
+            
+            Button(action: {
+                viewModel.addToMyBooks()
+            }) {
+                let image = viewModel.isTaken ? "star.fill" : "star"
+                Label(viewModel.isTaken ? "Added to My Books" : "Add to My Books", systemImage: image)
```

Comment:
```suggestion
                Label(viewModel.isTakenText, systemImage: viewModel.isTakenImage)
```
In MVVM business logic is good to be in view models. Maybe you can think of better names than isTakenText and isTakenImage, but the main point is that there shouldnt not be such logic in the view